app/src/main/java/de/westnordost/streetcomplete/quests/parcel_locker_brand/AddParcelLockerBrand.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.parcel_locker_brand
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddParcelLockerBrand : OsmFilterQuestType<String>() {
+
+    override val elementFilter = "nodes with amenity = parcel_locker and !brand and !name"
+    override val changesetComment = "Specify parcel locker brand"
+    override val wikiLink = "Tag:amenity=parcel_locker"
+    override val icon = R.drawable.ic_quest_notes
+    override val isDeleteElementEnabled = true
+    override val achievements = listOf(CITIZEN)
```

Comment:
Maybe this should rather count towards the `POSTMAN` achievement?

Reply:
The description of the quest is not very much in tune : `You helped to locate %d addresses!\nThis data is essential not only for the mail carrier but also for any navigation system.`

Reply:
Hmm good point. The `CITIZEN` achievement kind of turns into a "catch all" achievement though. Should we maybe expand the `POSTMAN` achievement to other post-related quests, and adjust its description accordingly? (Question to all)

Reply:
I think we can create a new achievement that can be used for parcel lockers and post partners.

https://wiki.openstreetmap.org/wiki/Tag:post_office%3Dpost_partner

Reply:
see also https://github.com/streetcomplete/StreetComplete/discussions/3009

Reply:
> create a new achievement

Note that this is relatively much effort, as you need to create a nice picture, find a good name and description, translate those texts, and have enough quests that count towards the new achievement.

See also #3009.

Reply:
> The description of the quest is not very much in tune : `You helped to locate %d addresses!\nThis data is essential not only for the mail carrier but also for any navigation system.`

Postman is also used for mailbox collection times, royal cypher and road name quests, so I think it would be ok to use here.

Reply:
Yeah, let's use POSTMAN and maybe change the description a bit. (Anyone wants to create a PR for that?)

Reply:
It's done ^^.